plugins/storage/server/api/files.test.ts

```diff
@@ -18,11 +21,33 @@ describe("#files.create", () => {
         key: "public/foo/bar/baz.png",
       },
     });
-    const body = await res.json();
     expect(res.status).toEqual(400);
-    expect(body.message).toEqual(
-      "key: Must be of the form uploads/<uuid>/<uuid>/<name> or public/<uuid>/<uuid>/<name>"
+  });
+
+  it("should fail with status 404 if existing file is requested with key", async () => {
```

Comment:
Misleading/incomplete text? This actually fails because of the missing attachment.

Reply:
I think it's ok, the file exists on disk and the key is valid but it returns 404 without any auth which is what I was aiming to test here.